Cargo.toml

```diff
@@ -17,6 +17,7 @@ build = "build.rs"
 libc = "0.2"
 nom = { version = "2", features = ["verbose-errors"] }
 byteorder = "0.5"
+time = "0.1"
```

Comment:
Hum, didn't read the whole review before posting that comment. So, I can replace it with a `Option<std::time::Duration>` if you meant to have a `None` for `unlimited` values